runtime/compiler/control/MethodToBeCompiled.hpp

```diff
@@ -61,14 +65,16 @@ struct TR_MethodToBeCompiled
    void acquireSlotMonitor(J9VMThread *vmThread);
    void releaseSlotMonitor(J9VMThread *vmThread);
    void setAotCodeToBeRelocated(const void *m);
+#if defined(JITSERVER_SUPPORT)
    bool isAotLoad() const { return _doAotLoad; }
    bool isRemoteCompReq() const { return _remoteCompReq; } // at the client
    void setRemoteCompReq() { _remoteCompReq = true; }
    void unsetRemoteCompReq() { _remoteCompReq = false; }
    bool isOutOfProcessCompReq() const { return _stream != NULL; } // at the server
    uint64_t getClientUID() const;
-   void cleanupJITaaS();
-   bool hasChangedToLocalSyncComp() const { return _origOptLevel != unknownHotness; }
+   void cleanupJITServer();
```

Comment:
I don't remember what `cleanupJITServer()` is supposed to do, so maybe a short comment would be useful.